### DISCUSSION TITLE
Is blobfuse2 supported on RHEL 8.8?

### DISCUSSION THREAD

**Discussion by japatel** [Azure/azure-storage-fuse#1568] [Q&A] [answered]
When I look at https://github.com/Azure/azure-storage-fuse/wiki/Blobfuse2-Supported-Platforms page I see support for RHEL 8.2. Question - Is blobfuse2 supported on RHEL 8.8?

**Comment by bhansalivikas** [comment, marked as answer]
RHEL 8 version shall be compatible so if you use 8.2 on 8.8 it shall just work fine.
